script.js

```diff
@@ -0,0 +1,78 @@
+// Selectors
+const btn = document.getElementById('btn');
+const output = document.getElementById('output');
+const container = document.querySelector('.container');
+
+
+// Events
+btn.addEventListener('click', showRandomMeal);
+output.addEventListener('click', showRandomMeal2);

```

Comment:
is this a button, i dont like the idea of calling the showrandomMeal function when someone clicks the page.

Reply:
Oh, the showRndomMeal() won't be invoked when the page is clicked. Just when the button is. I used an if statement to make sure of that.
And I only did that because for some reasons the initial styling on the page was absolutely positioning everything I was fetching from the api. So I had to create a new button dynamically and you can't add an event listener directly to a dynamically created element except through an event propagation with an element already existing and closely related to the element you want to add the event listener to. That was why I said I did a work around the issue I faced.